src/app/functions/server/onerep.ts

```diff
@@ -479,25 +438,16 @@ export async function getProfilesStats(
   if (profileStatsCache.has(queryParamsString))
     return profileStatsCache.get(queryParamsString);
 
-  const response: Response = await onerepFetch(
+  const profileStats = (await onerepFetch(
     `/stats/profiles?${queryParamsString}`,
     {
       method: "GET",
     },
-  );
-  if (!response.ok) {
-    logger.error(
-      `Failed to fetch OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-    throw new Error(
-      `Failed to fetch OneRep profile: [${response.status}] [${response.statusText}]`,
-    );
-  }
-
-  const profileStats: ProfileStats = await response.json();
+  )) as Promise<ProfileStats>;
 
   // cache results in map, with a flush hack to keep the size low
   if (profileStatsCache.size > 5) profileStatsCache.clear();
-  profileStatsCache.set(queryParamsString, profileStats);
+  profileStatsCache.set(queryParamsString, await profileStats);
   return profileStats;
 }
+export { getLatestOnerepScanResults };
```

Comment:
Why are we re-exporting this here? Doesn't it make sense to keep a distinction between modules that fetch data from the OneRep API, and those that fetch data from our own database?